templates/template/task_graph_classification.hjson

```diff
@@ -57,5 +62,19 @@
 
     # Key index of the saved npz data. (optional)
     "key": ""
-  }
-}
+  },
+
```

Comment:
Maybe add another line of "/* ------------------------------ Fixed splits ------------------------------ */" here to explicitly specify the begin and end?

Also, we may want to change the "required" and "optional" remark?